exasol/slc/internal/tasks/build/docker_build.py

```diff
@@ -1,48 +1,65 @@
+# pylint: disable=not-an-iterable
+from typing import Dict, Generator, Optional, Set, Tuple
+
 import luigi
+from exasol_integration_test_docker_environment.lib.base.abstract_task_future import (
+    AbstractTaskFuture,
+)
+from exasol_integration_test_docker_environment.lib.base.base_task import BaseTask
 from exasol_integration_test_docker_environment.lib.base.flavor_task import (
     FlavorsBaseTask,
 )
-from exasol_integration_test_docker_environment.lib.base.json_pickle_target import (
-    JsonPickleTarget,
-)
-from exasol_integration_test_docker_environment.lib.base.pickle_target import (
-    PickleTarget,
+from exasol_integration_test_docker_environment.lib.docker.images.image_info import (
+    ImageInfo,
 )
 from luigi import Config
-from luigi.format import Nop
 
 from exasol.slc.internal.tasks.build.docker_flavor_build_base import (
     DockerFlavorBuildBase,
 )
 
 
 class DockerBuildParameter(Config):
-    goals = luigi.ListParameter()
-    shortcut_build = luigi.BoolParameter(True)
+    goals: Tuple[str, ...] = luigi.ListParameter()  # type: ignore
```

Comment:
This is not nice from luigi. Even though the class is called `ListParameter`, the serialized object is a tuple. Needed to change this in several places...